test/auth.ts

```diff
@@ -9,16 +9,50 @@ import {JsonRpcAuth, utils} from './../src/'
 const {jsonRequest} = utils
 
 describe('Auth', function() {
-    this.slow(3 * 1000)
-    this.timeout(5 * 1000)
+    this.slow(1000)
```

Comment:
i would remove this now, don't think you need it anymore

Reply:
Tests where signing is involved takes ~200ms on my machine. Want me to profile it?

Reply:
that should be well within the default test timeout period though, no?

Reply:
Probably but mocha will flag them as slow

Reply:
ok disregard